src/integration/byte_mappings.py

```diff
@@ -24,6 +24,201 @@
     0x1A: "Dragon",
 }
 
+
+GEN_1_SPECIES = [
+    "",
+    "Rhydon",
+    "Kangaskhan",
+    "Nidoran",
+    "Clefairy",
+    "Spearow",
+    "Voltorb",
+    "Nidoking",
+    "Slowbro",
+    "Ivysaur",
+    "Exeggutor",
+    "Lickitung",
+    "Exeggcute",
+    "Grimer",
+    "Gengar",
+    "Nidoran",
+    "Nidoqueen",
+    "Cubone",
+    "Rhyhorn",
+    "Lapras",
+    "Arcanine",
+    "Mew",
+    "Gyarados",
+    "Shellder",
+    "Tentacool",
+    "Gastly",
+    "Scyther",
+    "Staryu",
+    "Blastoise",
+    "Pinsir",
+    "Tangela",
+    "MissingNo",
+    "MissingNo",
+    "Growlithe",
+    "Onix",
+    "Fearow",
+    "Pidgey",
+    "Slowpoke",
+    "Kadabra",
+    "Graveler",
+    "Chansey",
+    "Machoke",
+    "Mr",
```

Comment:
Mr who 👀

Reply:
Curses. I caught Farfetch'd, but missed this 😁